docs/blocks-storage/querier.md

```diff
@@ -0,0 +1,545 @@
+---
```

Comment:
Maybe add that this file is generated from the `<path-to-template>.template`?

Reply:
Neat idea! Done